airbyte-integrations/connectors/source-oracle/src/test-integration/java/io/airbyte/integrations/source/oracle/OracleSourceDatatypeTest.java

```diff
@@ -14,6 +14,7 @@
 import io.airbyte.integrations.standardtest.source.TestDataHolder;
 import io.airbyte.integrations.standardtest.source.TestDestinationEnv;
 import io.airbyte.protocol.models.JsonSchemaType;
+import java.math.BigDecimal;
```

Comment:
Is this being used anywhere? Unless the diff doesn't show the changes that use this import